### PR TITLE
[ACI-1586] Add new inputs

### PR DESCRIPTION
**Pull request by lpusok**
Added debug, validation level, pus hand endpoint inputs.

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -23,3 +23,4 @@ linters:
   - testpackage
   - mnd
   - gomnd
+  - err113 # ensble use of dynamic errors fmt.Errorf("error: %s", str)
```
Comment (zsolt-marta-bitrise) [inline .golangci.yaml:26]: Why do you need `fmt.Errorf` for those cases, and why not use `fmt.Sprintf`? Errorf should be used for `fmt.Errorf("call api: %w", err)`

File: internal/config/gradle/gradleconfig_test.go [hunk 2]
```diff
@@ -77,9 +87,14 @@ func TestGenerateInitGradle(t *testing.T) {
...
-				endpointURL:         "grpcs://remote-build-cache.services.bitrise.io",
-				authToken:           "AuthT0ken",
-				metricsEnabled:      true,
+				endpointURL: "grpcs://remote-build-cache.services.bitrise.io",
+				authToken:   "AuthT0ken",
+				userPrefs: Preferences{
+					IsPushEnabled:        true,
+					CacheLevelValidation: CacheValidationLevelWarning,
+					IsAnalyticsEnabled:   true,
+					IsDebugEnabled:       true,
```
Comment (zsolt-marta-bitrise) [inline internal/config/gradle/gradleconfig_test.go:96]: Please add a test case for debug = false and push = false

File: cmd/enableForGradle.go [hunk 3]
```diff
@@ -71,9 +80,18 @@ func enableForGradleCmdFn(logger log.Logger, gradleHomePath string, envProvider
 
 	// Optional configs
 	// EndpointURL
-	endpointURL := common.SelectEndpointURL(envProvider("BITRISE_BUILD_CACHE_ENDPOINT"), envProvider)
+	endpointURL := common.SelectEndpointURL(paramRemoteCacheEndpoint, envProvider)
 	logger.Infof("(i) Build Cache Endpoint URL: %s", endpointURL)
+	logger.Infof("(i) paramIsPushEnabled: %t", paramIsPushEnabled)
+	logger.Infof("(i) paramValidationLevel: %s", paramValidationLevel)
 	logger.Infof("(i) paramIsGradleMetricsEnabled: %t", paramIsGradleMetricsEnabled)
+	logger.Infof("(i) isDebugLogMode: %t", isDebugLogMode)
```
Comment (zsolt-marta-bitrise) [inline cmd/enableForGradle.go:88]: Where is this populated?

File: .golangci.yaml (hunk 1 above)
Comment (lpusok) [reply .golangci.yaml:26]: This linter also prevents  `errors.New("")` for example.

File: cmd/enableForGradle.go (hunk 3 above)
Comment (lpusok) [reply cmd/enableForGradle.go:88]: Here: https://github.com/bitrise-io/bitrise-build-cache-cli/blob/38f5096b3ce65f6637a2eb996001c61b006e23ca/cmd/root.go#L34

File: .golangci.yaml (hunk 1 above)
Comment (zsolt-marta-bitrise) [reply .golangci.yaml]: I seriously doubt that 🤔 Are you sure?

**Review by zsolt-marta-bitrise — APPROVED** [review]

File: .golangci.yaml (hunk 1 above)
Comment (zsolt-marta-bitrise) [reply .golangci.yaml]: I saw you reverted this, so approving

**Pull request merged**